sql/adhoc.sql

```diff
@@ -187,7 +187,8 @@ SELECT
         SUM(settle_amount)::numeric as total,
         DATE(timestamp) as day
       FROM stripe
-      WHERE timestamp > '2015-11-01'
+      WHERE timestamp > '2015-10-04T07:00:00.000Z'
+      AND status = 'succeeded'
```

Comment:
Hah! I totally missed that when I pulled from master.

We can ditch this PR, the timestamp isn't needed as I only need to look at the last couple of days each time I run this.